builder/internals.go

```diff
@@ -90,6 +90,16 @@ func (b *Builder) commit(id string, autoCmd []string, comment string) error {
 			return err
 		}
 		defer container.Unmount()
+
+		// Make sure all resources are setup, but we can ignore errors
+		// Supress output from this
```

Comment:
Specifically I was seeing an error in the tests where:
```Dockerfile
FROM scratch
MAINTAINER jpetazzo@docker.com
```
Since `MAINTAINER` the maintainer line is trying to be run with `/bin/sh`, but `/bin/sh` doesn't exist in `scratch`